lib/rbac/filterer.rb

```diff
@@ -110,6 +110,7 @@ class Filterer
       'MiqRequest'             => :descendant_ids,
       'MiqRequestTask'         => nil, # tenant only
       'MiqTemplate'            => :ancestor_ids,
+      'OrchestrationStack'     => nil,
```

Comment:
Seems templates / generic objects tend to be `ancestor_ids`,
and core objects tend to be `descendant_ids`.

I expected this one to be `descendant_ids`

Reply:
ok, can we remove this?

a. I think this array is only for tenancy and not cloud tenancy
b. Having a `nil` in here will not do anything

Also, can you verify that we are not calling into `accessible_tenant_ids_strategy` for `OrchestrationStack`?
I'm pretty sure we don't need that

Reply:
let me make clear those things... 

> a. I think this array is only for tenancy and not cloud tenancy

It works togethers.  We need find ids from tenants and then find their equivalents in cloud tenants.

in filterer:
1) it goes to [scope_to_cloud_tenant](https://github.com/ManageIQ/manageiq/blob/e93bf1d99c6c9e9128b88cb7d2944993166d0e3d/lib/rbac/filterer.rb#L575)

2) then to the [TenancyCommonMixin#tenant_id_clause](https://github.com/ManageIQ/manageiq/blob/a68ba141c77af1d728615525d5c2888376eaa51a/app/models/mixins/tenancy_common_mixin.rb#L9)

3) and then in TenancyCommonMixin#tenant_id_clause it calls [tenant_id_clause_format](https://github.com/ManageIQ/manageiq/blob/ab09c7ca3163db40574d7f1a1121c02a64da8648/app/models/mixins/cloud_tenancy_mixin.rb#L16)  but from **CloudTenancyMixin**


> b. Having a `nil` in here will not do anything
so form previous answer (2) **it is needed to have `nil` here -** it means "current" strategy (it is considering just tenant from logged user - no ancestors or descendants)


> Also, can you verify that we are not calling into accessible_tenant_ids_strategy for OrchestrationStack?
I'm pretty sure we don't need that

Yes I verified it when I was looking for previous answers. `accessible_tenant_ids_strategy ` is called also from `OrchestrationStack ` and it is called [here](https://github.com/ManageIQ/manageiq/blob/a68ba141c77af1d728615525d5c2888376eaa51a/app/models/mixins/tenancy_common_mixin.rb#L10)


@kbrock does it make sense now ? If you other questions let me know!

thanks!